stdlib/public/core/Reverse.swift

```diff
@@ -312,7 +312,7 @@ extension LazyCollectionProtocol
   ///
   /// - Complexity: O(1)
   @_inlineable
-  public func reversed() -> LazyBidirectionalCollection<
+  public func reversed() -> LazyCollection<
```

Comment:
The `extension LazyCollectionProtocol where Self : RandomAccessCollection` can go now, right?

Reply:
Ouch, this change is wrong. I did not merge all the `Lazy${Traversal}Collection`s into one. Only the `Filter` and `Map` ones.